src/functionalTest/resources/application-functional.yaml

```diff
@@ -7,7 +7,7 @@ generated.user.email.pattern: ${GENERATED_USER_EMAIL_PATTERN:freg-test-user-%s@f
 test.url: ${TEST_URL:http://rd-user-profile-api-preview.service.core-compute-preview.internal}
 test.user.password: ${TEST_USER_PASSWORD:Hmcts123}
```

Comment:
can you check idam.auth.tokenAuth can be removed ? i think this will come from vault.

Reply:
removed entire values which are not required